domains/kokofixcomputers.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kokofixcomputers",
+        "email": "github@kokofixcomputers.serv00.net"
+    },
+    "record": {
+        "TXT": "013e49fe981141dd118a0148d40980"
```

Comment:
This record would need to be on a nested subdomain starting with `_github-pages-challenge` and you would need to have a record on `kokofixcomputers.is-a.dev` pointing to your GitHub Pages website.

Reply:
Huh? How?